src/main/groovy/org/jenkinsci/gradle/plugins/jpi/internal/ConfigureUtil.groovy

```diff
@@ -0,0 +1,32 @@
+package org.jenkinsci.gradle.plugins.jpi.internal
+
+import org.codehaus.groovy.runtime.GeneratedClosure
+import org.gradle.internal.metaobject.ConfigureDelegate
+import org.gradle.util.Configurable
+import org.gradle.util.internal.ClosureBackedAction
+import javax.annotation.Nullable
+
+class ConfigureUtil {
```

Comment:
The plugin might need some changes to stop doing this and be more idiomatic but didn't want to introduce more moving pieces. Instead went for introducing the current approach.

Also it seems that the usages come from deprecated classes